web/themes/joinup/templates/user/user--group-header.html.twig

```diff
@@ -50,6 +50,12 @@
             <span class="stats__text">{{ 'Member for'|t }}:</span>
             <span class="stats__text bold">{{ member_for }}</span>
           </div>
+          {% if content.field_user_nationality|render %}
```

Comment:
Do we need the render function here? I tried it and it works without the render function. If you remove it from the view mode, it does not show, it does not log any errors. But here we are rendering it once more.
Let's remove it (I know it is like this in the rest) if there is no reason for it.

Reply:
Really!
Changed!